### ISSUE TITLE
Location data incorrect 

### ISSUE THREAD

**Issue by pjmurray** [koshilife/calendly-api-ruby-client#18]
https://github.com/koshilife/calendly-api-ruby-client/blob/951ce8138ae9d404cb6883417f5de6eb549746a0/lib/calendly/models/location.rb#L43

- [ ] `type` not `kind`
- [ ] expose `join_url`  form zoom
- [ ] `status` not state

As per https://developer.calendly.com/docs/api-docs/reference/calendly-api/openapi.yaml/paths/~1scheduled_events~1%7Buuid%7D/get

**Comment by koshilife** [comment]
Hi @pjmurray,

Thank you for issue again.
At 2020 August I asked Calendly API teams same thing, it was undergoing.
There may have not been solved yet.

Could you share the api request/response data?
You can get respose data in your console like below.

```ruby
Calendly.configuration.logger.level = :debug
```

**Comment by pjmurray** [comment]
Thanks!
Will do when I get to it! Right now I am just doing the integration via
automated testing :)

Hoping they issue me some dev/staging keys shortly so I'm unblocked!

PJ

**Comment by koshilife** [comment]
@pjmurray 

I'm sorry :)
I had misunderstood, your pointing are right!
I didn't know that the Location object format had changed such as "kind" field to "type" field.

I checked the Location object has been returned Calendly v2 API.
I am going to deal with it in near days.

**Comment by koshilife** [comment]
Hi @pjmurray,

I released fixed version '0.4.0'.
Could you check this issue has been solved?

```
gem 'calendly', '>= 0.4.0'
```

like following
```ruby
>> Calendly::VERSION
=> "0.4.0"
>> client = Calendly::Client.new 'TOKEN'
>> ev = client.me.scheduled_events.last
>> ev.location.type
=> "zoom"
>> ev.location.join_url
=> "https://us04web.zoom.us/j/123456789?pwd=NVRRSGRYcW52OHlMMGlXNmpYWHQrQT09"
>> ev.location.data
=> {:id=>123456789, :settings=>{}, :password=>"NS0dc7", :extra=>nil}
```

**Comment by pjmurray** [comment]
Great this is working as expected thank you
